### ISSUE TITLE
Repeating Newton gives :unknown

### ISSUE THREAD

**Issue by dpsanders** [JuliaIntervals/IntervalRootFinding.jl#29]
It should keep track of the `:unique` information.
```
julia> rts = roots(x->x^2 - 2, -10..10)
2-element Array{IntervalRootFinding.Root{IntervalArithmetic.Interval{Float64}},1}:
 Root(Interval(1.414213562373095, 1.4142135623730951), :unique)
 Root(Interval(-1.4142135623730951, -1.414213562373095), :unique)

julia> roots(x->x^2 - 2, rts)
2-element Array{IntervalRootFinding.Root{IntervalArithmetic.Interval{Float64}},1}:
 Root(Interval(1.414213562373095, 1.4142135623730951), :unknown)
 Root(Interval(-1.4142135623730951, -1.414213562373095), :unknown)
```

**Comment by lbenet** [comment]
The question is, why do we get a different result?

**Comment by dpsanders** [comment]
The interval is already atomic, so it can't get any smaller, so the test in Newton that the result should be a strict subset fails.

**Comment by dpsanders** [comment]
One solution may be to expand it slightly and try again. But that has the danger of possibly introducing a new root.

**Comment by dpsanders** [comment]
e.g. for an interval `II` do something like (in 1D) 
```
 II = mid(II) ± 2radius(II)
```

**Comment by lbenet** [comment]
What about checking the atomomicity of the interval? If the interval is atomic, do not change the already obtain `status` of the root. 

Something like this:
```julia
julia> a = Interval(1.414213562373095, 1.4142135623730951) # first root above
[1.41421, 1.41422]

julia> diam(a) <= eps(a) # check if the interval is as thin as it can get
true
```

**Comment by dpsanders** [comment]
Fixed by #97!
